src/editor/NoteContent.js

```diff
@@ -1,28 +1,25 @@
 import * as React from "react";
 import {Note} from "../model/Note";
+import {useDispatch} from "react-redux";
+import {updateNote} from "../actions";
 
```

Comment:
Extract the props directly in the head of the function so that you can fast get to know everything that is needed to create it.
`
export const NoteContent = function ({activeNote}) {
...
`

Reply:
Yup, that's nice technique. I'll do it later. After all I have plans to rewrite it to tsx, so props probably will become some kind of interface or inline type.